apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -353,6 +353,10 @@ suite('ActivityHandler', function() {
         this.sinon.stub(Contacts, 'findByAddress').returns(findByPromise);
 
         MockNavigatormozApps.mTriggerLastRequestSuccess();
+
+        findByPromise.then(() => {
```

Comment:
Another idea is return promise in success request, and we can handle the readiness in `MockNavigatormozApps.mTriggerLastRequestSuccess().then()`(we will need to return the success in mock as well). But not sure if it's safe for all the tests.